tests/ui/landing_page_test.py

```diff
@@ -24,6 +24,12 @@ def setUp(self):
   def tearDown(self):
     """Teardown for test methods."""
     landing_page = LandingPage(self.driver)
+    try:
+      user_details_button = landing_page.get_element(
```

Comment:
Wish I have a snapshot of this failure to see why it can not click.  Do you think we can make the get_element() more robust, such as with an expected condition?
https://github.com/uProxy/ufo-management-server-flask/blob/7bb7d38860435400f402782c7b87162d2ab950a8/tests/ui/base_driver.py#L22